move/its/sources/its.move

```diff
@@ -1612,229 +1598,25 @@ fun test_set_trusted_address() {
     let ctx = &mut tx_context::dummy();
     let mut its = create_for_testing(ctx);
 
-    let trusted_source_chain = ascii::string(b"Axelar");
-    let trusted_source_address = ascii::string(b"Trusted Address");
-    let message_type = (123 as u256);
-    let message_id = ascii::string(b"message_id");
-
-    let governance = governance::new_for_testing(
-        trusted_source_chain,
-        trusted_source_address,
-        message_type,
+    let owner_cap = owner_cap::create(
         ctx,
     );
 
-    let trusted_chains = vector[b"Ethereum", b"Avalance", b"Axelar"];
+    let trusted_chains = vector[b"Ethereum".to_ascii_string(), b"Avalance".to_ascii_string(), b"Axelar".to_ascii_string()];
```

Comment:
```suggestion
    let trusted_chains = vector[String(b"Ethereum"), String(b"Avalanche"), String(b"Axelar")];
```
```suggestion
    let trusted_chains = vector[b"Ethereum", b"Avalanche", b"Axelar"].map!(|chain| String(chain));
```
use a map! to convert or just import String

Reply:
done